doc/rst/meta/modules/standard.rst

```diff
@@ -6,19 +6,111 @@ Standard Modules
 Standard modules are those which describe features that are considered
 part of the Chapel Standard Library.
 
+
+Automatic Modules
+-----------------
+
 All Chapel programs automatically ``use`` the modules :chpl:mod:`Builtins` ,
 :chpl:mod:`Math` , and :chpl:mod:`Types` by default.
 
+.. toctree::
+   :maxdepth: 1
+
+   Builtins <standard/Builtins>
+   Math <standard/Math>
+   Types <standard/Types>
+
+
+Data Structures
+---------------
+
+.. toctree::
+   :maxdepth: 1
+
+   Heap <standard/Heap>
+   List <standard/List>
+   Map <standard/Map>
+   Set <standard/Set>
+
+   
+Diagnostics
+-----------
+
+.. toctree::
+   :maxdepth: 1
+
+   CommDiagnostics <standard/CommDiagnostics>
+   Memory <standard/Memory>
+   Time <standard/Time>
+
+   
+Files/IO
+--------
+.. toctree::
+   :maxdepth: 1
+
+   FileSystem <standard/FileSystem>
+   IO <standard/IO>
+   Path <standard/Path>
+   Regexp <standard/Regexp>
```

Comment:
This one made me pause.

Arguably, regular expressions are commonly used alongside file/IO operations, but they are not limited to that.

Reply:
I wasn't crazy about it either, but it was better than my first category for it ("Misc").  :)

Reply:
I put this under the new Utilities category.